apps/emqx_bridge_gcp_pubsub/test/emqx_bridge_gcp_pubsub_SUITE.erl

```diff
@@ -66,6 +67,7 @@ single_config_tests() ->
     ].
 
 init_per_suite(Config) ->
+    emqx_common_test_helpers:clear_screen(),
```

Comment:
I'd strongly vote for not doing such things with dev's console. 😅

Reply:
Oops, forgot to remove it before committing... :see_no_evil:

Reply:
:heavy_check_mark: